sky/provision/fluidstack/instance.py

```diff
@@ -101,25 +101,33 @@ def run_instances(region: str, cluster_name_on_cloud: str,
     exist_instances = _filter_instances(cluster_name_on_cloud, ['running'])
     head_instance_id = _get_head_instance_id(exist_instances)
 
-    def rename(instance_id, new_name):
+    def rename(instance_id: str, new_name: str) -> None:
```

Comment:
Hmm, I think it's a bit weird we have to do it for Fluidstack but not other clouds. Renaming seems to assume reusing existing VMs is ok. Is it right to say an alternative is to down those old instances and relaunch everything? If so, I think it's fine to leave a NOTE/TODO and unblock the user first.